src/components/BlockchainList/PostsList.tsx

```diff
@@ -1,34 +1,80 @@
-import { Suspense } from 'preact/compat'
+import { AccentText, LoadingText } from 'components/Text'
+import { Suspense, useState } from 'preact/compat'
 import { useSnapshot } from 'valtio'
 import BlockchainPost from 'components/BlockchainList/BlockchainPost'
+import InfiniteScroll from 'react-infinite-scroll-component'
 import ListLoading from 'components/ListLoading'
 import NoPosts from 'components/BlockchainList/NoPosts'
 import PostStore from 'stores/PostStore'
+import SealLogo from 'icons/SealLogo'
+import classnames, {
+  alignItems,
+  display,
+  flexDirection,
+  gap,
+  margin,
+  width,
+} from 'classnames/tailwind'
 import useScrollToAnchor from 'hooks/useScrollToAnchor'
 
+const endMessageWrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  alignItems('items-center'),
+  margin('my-4'),
+  gap('gap-y-2'),
+  width('w-full')
+)
+
 function BlockchainPostsListSuspended() {
-  const { selectedPosts, selectedToken } = useSnapshot(PostStore)
+  const { selectedPosts } = useSnapshot(PostStore)
   useScrollToAnchor()
 
+  const data = selectedPosts.filter((post) =>
+    PostStore.selectedToken
+      ? post.derivativeAddress === PostStore.selectedToken
+      : post
+  )
+  const EndMessage = () => (
```

Comment:
Do we need `endmessage` actually? The user will see endmessage + footer, maybe it's better to display only footer?

Reply:
I don't think that it's a bad idea to show the end message. But, of course, it can be removed.

Reply:
But it still needs to be defined even as an empty block

Reply:
Removed `EndMessage` and added an empty fragment instead